interface/patient_file/history/history.php

```diff
@@ -76,24 +81,41 @@
                 $result = getHistoryData($pid);
             }
             ?>
-
-            <?php if (acl_check('patients', 'med', '', array('write','addonly'))) { ?>
-                <div class="page-header">
-                    <h2><?php echo htmlspecialchars(getPatientName($pid), ENT_NOQUOTES);?> <small><?php echo xl("History & Lifestyle");?></small></h2>
-                </div>
-                <div>
-                    <div class="btn-group">
-                        <a href="../summary/demographics.php" class="btn btn-default btn-back" onclick="top.restoreSession()">
-                            <?php echo htmlspecialchars(xl('Back To Patient'), ENT_NOQUOTES);?>
-                        </a>
-                        <a href="history_full.php" class="btn btn-default btn-edit" onclick="top.restoreSession()">
-                            <?php echo htmlspecialchars(xl("Edit"), ENT_NOQUOTES);?>
-                        </a>
-                    </div>
+        </div>
+    </div>
+    <?php
+    if (acl_check('patients', 'med', '', array('write','addonly'))) {
+        $header_title = xl('History and Lifestyle of');?>
+        <div class="row">
+            <div class="col-sm-12">
+                <?php
+                $portal_login_href = "../summary/create_portallogin.php";
```

Comment:
above $portal_login_href  is not necessary since we now know how to use the webroot path in the dashboard_header.php script :)

Reply:
Nice catch, one more redundant line of code removed. I have added 
```
$url_webroot = $GLOBALS['webroot'];
$portal_login_href = $url_webroot ."/interface/patient_file/summary/create_portallogin.php";
```
to `dashboard_header.php` and eliminated the relative path to `summary/create_portallogin.php` from 
```
interface/patient_file/history/history.php
interface/patient_file/summary/demographics.php
interface/patient_file/summary/stats_full.php
interface/patient_file/report/patient_report.php
interface/patient_file/transaction/transactions.php
interface/reports/external_data.php
interface/reports/pat_ledger.php
```

Using the same principle I have eliminated all the relative paths in the anchor tags in `header.php` :)

I have updated the PR with all of the above changes plus the changes that you have suggested